### ISSUE TITLE
preferences not being loaded

### ISSUE THREAD

**Issue by preetam-kotteda** [vercel/hyper#5706]
- [x] Your Hyper.app version is **3.0.2**. Please verify your using the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
  - [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

  ---
  - **Any relevant information from devtools?** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_:

  - **Is the issue reproducible in vanilla Hyper.app?**

## Issue

 - **Hyper version**: undefined "3.0.2"

 - **OS ARCH VERSION:** win32 x64 10.0.19042
 - **Electron:** 3.1.3  **LANG:** undefined
 - **SHELL:** undefined   **TERM:** undefined

  <details>
    <summary><strong> ~/.hyper.js contents</strong></summary>
      <pre>
        <code>
          {
  "updateChannel": "stable",
  "fontSize": 12,
  "fontFamily": "Menlo, \"DejaVu Sans Mono\", Consolas, \"Lucida Console\", monospace",
  "fontWeight": "normal",
  "fontWeightBold": "bold",
  "lineHeight": 1,
  "letterSpacing": 0,
  "cursorColor": "rgba(248,28,229,0.8)",
  "cursorAccentColor": "#000",
  "cursorShape": "BLOCK",
  "cursorBlink": false,
  "foregroundColor": "#fff",
  "backgroundColor": "#000",
  "selectionColor": "rgba(248,28,229,0.3)",
  "borderColor": "#333",
  "css": "",
  "termCSS": "",
  "showHamburgerMenu": "",
  "showWindowControls": "",
  "padding": "12px 14px",
  "colors": {
    "black": "#000000",
    "red": "#C51E14",
    "green": "#1DC121",
    "yellow": "#C7C329",
    "blue": "#0A2FC4",
    "magenta": "#C839C5",
    "cyan": "#20C5C6",
    "white": "#C7C7C7",
    "lightBlack": "#686868",
    "lightRed": "#FD6F6B",
    "lightGreen": "#67F86F",
    "lightYellow": "#FFFA72",
    "lightBlue": "#6A76FB",
    "lightMagenta": "#FD7CFC",
    "lightCyan": "#68FDFE",
    "lightWhite": "#FFFFFF"
  },
  "shell": "",
  "shellArgs": [
    "--login"
  ],
  "env": {},
  "bell": "SOUND",
  "copyOnSelect": false,
  "defaultSSHApp": true,
  "quickEdit": false,
  "macOptionSelectionMode": "vertical",
  "webGLRenderer": true
}

          {
  "plugins": [],
  "localPlugins": []
}
        </code>
      </pre>
  </details>

**Comment by preetam-kotteda** [comment]
My hyper preferences not getting updated

**Comment by obouchari** [comment]
Same here It doesn't load preferences. I'm on Mac M1, Big sure, Hyper version 3.1.2 (stable).

**Comment by wguser13** [comment]
same issue here!

**Comment by LabhanshAgrawal** [comment]
Can you give the output from `hyper -v` and from the developer console (cmd+alt+i)
@preetam-kotteda can you post the config that you're trying to use? This looks like the default one.

**Comment by preetam-kotteda** [comment]
Thank you for your response issue got resolved.

**Comment by LabhanshAgrawal** [comment]
@wguser13 can you check?

**Comment by wguser13** [comment]
@LabhanshAgrawal I am new to this. I typed the command you told me (hyper -v) but nothing came up. I have included screenshot of Developer Counsel and link that it takes me to in the browser when i click on preferences.
![1](https://user-images.githubusercontent.com/93053410/142356586-9d075520-0e65-4745-9e36-f67984550424.png)
<img width="940" alt="2" src="https://user-images.githubusercontent.com/93053410/142356587-3d652bfb-e698-4793-9062-1b4c789f05a0.png">
